test/trailing_spaces_br.md

```diff
@@ -1,4 +1,6 @@
 This line has a single trailing space {MD009} 
 This line has two trailing spaces and should be allowed  
 This line has three trailing spaces {MD009}   
+ 
+{MD009:4}
```

Comment:
I love this new test case! Can you please move it to the end instead of inside the 1/2/3/4 examples?

Reply:
👍